python/verifiers/ransac.py

```diff
@@ -0,0 +1,28 @@
+
+import cv2
+
+from verifiers.VerificationTemplate import VerificationTemplate
+
+class RANSAC(VerificationTemplate):
+
+    def __init__(self):
+        super(
+            RANSAC,
+            self).__init__(
+                name='RANSAC',
+                estimates_essential=True,
+                estimates_fundamental=True)
+
+    def evaluate_matches(self, pts1, pts2):
+        _, mask = cv2.findEssentialMat(pts1, pts2, method=cv2.RANSAC)
```

Comment:
@albutko same story here. a good example is https://github.com/argoai/argoverse-api/blob/master/argoverse/utils/cv2_plotting_utils.py#L21